### ISSUE TITLE
Check for parrallel plugin availability trough multiple locations

### ISSUE THREAD

**Issue by JAuriac** [pdidev/pdi#500]
Currently, finding a non-parallel NET-CDF plugin through ccmake AUTO prevents finding a parrallel NET-CDF elsewhere (where EMBEDDED could have found a suitable one). It may be better to enable this with a warning.

**Comment by JAuriac** [comment]
In a general manner and not for a quick fix, it is not advised to have automated behaviours in cmake which would by themselves search the system for another version (anti-pattern).

Changing the scope of this issue to instead modify the error message (of the root CMakeLists) to account for the hardcoded version of NetCDF (of NetCDF's CMakeLists), as we can have either an issue of missing parallel feature or version mismatch, similarly to HDF5.

**Comment by jbigot** [comment]
Yes, we do not build embedded version of libraries if a system version is available by default as this can break a lot of things. We leave it to the user to explicitly make this choice. This should be better explained in the error message as you note it.
